### ISSUE TITLE
Has anyone tried monterey? 

### ISSUE THREAD

**Issue by tylermmoore** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#64]
Trying it now but having some issues.

**Comment by Xtendera** [comment]
Yes, indeed I have. I don't seem to be having any issues, can you please tell me what SMBIOS you are using, your boot flags, and what specific issues you are having.

Cheers,
Ashray

**Comment by strikersix23** [comment]
Hey can you send me a guide for monterey

**Comment by Xtendera** [comment]
https://dortania.github.io/OpenCore-Install-Guide/

**Comment by Scott-GT** [comment]
Yes and it works very...very well. Modifications are required. I've put a note in the general area with the basic steps I took. 

The updates also resolved an issue I had with the screen not responding after going to sleep. It now sleeps and wakes perfectly.
